### PR TITLE
✨ Add TitleElement

### PR DESCRIPTION
**Pull request by manusmd**
# What I did
- Created TitleElement
- Contains grid with following content:
  - app logo
  - dynamic title
  - account logo

# Design
image

Part of #4

### CODE REVIEW

**Review by derjubu — APPROVED** [review]
LGTM :-)

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.stories.tsx":"1","/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.tsx":"2"},{"size":190,"mtime":1638179238269,"results":"3","hashOfConfig":"4"},{"size":343,"mtime":1638179238269,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"17owpm0",{"filePath":"8","messages":"9","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.stories.tsx",[],"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.tsx",[]]
```
Comment (lajanzen) [inline .eslintcache:1]: You could at this file to `.gitignore`.

**Review by lajanzen — APPROVED** [review]
🚀

File: .eslintcache (hunk 1 above)
Comment (manusmd) [reply .eslintcache:1]: I pushed this change to main branch. So it should be solved now.

**Pull request merged**